srtcore/core.cpp

```diff
@@ -4621,6 +4820,7 @@ void CUDT::acceptAndRespond(const sockaddr* peer, CHandShake* hs, const CPacket&
    m_iRcvLastSkipAck = m_iRcvLastAck;
    m_iRcvLastAckAck = hs->m_iISN;
    m_iRcvCurrSeqNo = hs->m_iISN - 1;
+   m_iRcvCurrPhySeqNo = hs->m_iISN - 1;
```

Comment:
"Phy" for packet filter?

Reply:
See core.cpp around line 8247. This is the latest sequence number of the packet really received. Introduction of the packet filter has turned those counters that were used so far for the physical sequence received (because there wasn't any other) into logical one, that is, possibly increased with the filter-rebuilt packet. This would hide the real loss number from statistics, where the lost packet must be notified always, regardless of the recovery method.